tests/System/Extension/ClassToFile/Rpc/FileInfoTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Phpactor\Tests\System\Extension\ClassToFile\Rpc;
+
+use Phpactor\Tests\System\SystemTestCase;
+
+class FileInfoTest extends SystemTestCase
```

Comment:
This one is not necessary I think, at least there is no precedent for RPC system tests at the moment, so reluctant to duplicate effort here.

The test I suggested was for the VIM plugin, which would have been an E2E test. These are few and are kept in `plugin/tests/*`, this one could look like:

```
# plugin/tests/get_namespace.vader
Execute:
    execute "edit src/Phpactor.php"
    namespace = call phpactor#GetNamespace()
    AssertEqual "Phpactor", namespace
```

Like I said, you would need the [vader](https://github.com/junegunn/vader.vim) plugin to execute this (with `:Vader` from within VIM with that file open).

But, if you don't want to do this, I can put it in later. No worries.